jans-client-api/server/src/main/java/io/jans/ca/server/configuration/AppInitializer.java

```diff
@@ -0,0 +1,216 @@
+                logger.info("Trató de leer: {}", entryManager);
+            } catch (Exception e) {
+                logger.warn("Unable to create persistence entry manager, retrying in {} seconds", RETRY_INTERVAL);
+                Thread.sleep(RETRY_INTERVAL * 1000L);
+            }
+        } while (entryManager == null && i < RETRIES);
+
+        if (entryManager == null) {
+            logger.error("No EntryManager could be obtained");
+        }
+        return entryManager;
+
+    }
+
+    public void recreatePersistanceEntryManager(@Observes @LdapConfigurationReload String event) {
+        closePersistenceEntryManager();
+        PersistenceEntryManager ldapEntryManager = persistenceEntryManagerInstance.get();
+        persistenceEntryManagerInstance.destroy(ldapEntryManager);
+        logger.debug("Recreated instance {} with operation service: {} - event:{}", ldapEntryManager,
+                ldapEntryManager.getOperationService(), event);
+    }
+
+    private void closePersistenceEntryManager() {
+        PersistenceEntryManager oldInstance = CdiUtil.getContextBean(beanManager, PersistenceEntryManager.class,
+                ApplicationFactory.PERSISTENCE_ENTRY_MANAGER_NAME);
+        if (oldInstance == null || oldInstance.getOperationService() == null)
+            return;
+
+        logger.debug("Attempting to destroy {} with operation service: {}", oldInstance,
+                oldInstance.getOperationService());
+        oldInstance.destroy();
+        logger.debug("Destroyed {} with operation service: {}", oldInstance, oldInstance.getOperationService());
+    }
+
+    public void destroy(@Observes @BeforeDestroyed(ApplicationScoped.class) ServletContext context) {
+        logger.info("================================================================");
+        logger.info("===========  jans-client-api service STOPPED  ==========================");
+        logger.info("servletContext:{}", context);
+
+        logger.info("================================================================");
+    }
+
+    protected void initSchedulerService() {
+        logger.debug("Initializing Scheduler Service");
+        quartzSchedulerManager.start();
+
+        String disableScheduler = System.getProperties().getProperty("gluu.disable.scheduler");
+        if (Boolean.parseBoolean(disableScheduler)) {
+            this.logger.warn("Suspending Quartz Scheduler Service...");
+            quartzSchedulerManager.standby();
+        }
+    }
+
+    private void clearRPTestData() {
+        try {
+            String val = System.getProperty("clearTestData");
+            if (val != null && !val.isEmpty() && Boolean.valueOf(val).booleanValue()) {
+                persistenceService.create();
+                rpService.removeAllRps();
+                rpService.load();
+                logger.info("Finished removeExistingRps successfullly.");
+            } else {
+                logger.info("Invalid value clearTestData.");
```

Comment:
lets not print it because normally operational jans-client-api should not have that property